lib/cdo/db.rb

```diff
@@ -47,3 +47,4 @@ def sequel_connect(writer, reader, validation_frequency: nil)
 PEGASUS_DB = sequel_connect CDO.pegasus_db_writer, CDO.pegasus_db_reader
 POSTE_DB = PEGASUS_DB
 DASHBOARD_DB = sequel_connect CDO.dashboard_db_writer, CDO.dashboard_db_reader
+DASHBOARD_DB_READER = sequel_connect CDO.dashboard_db_reader, CDO.dashboard_db_reader
```

Comment:
This is not what you want - this will be read-only against the production DB, not the reporting DB. Think you want:
`DASHBOARD_REPORTING_DB_READER = sequel_connect CDO.dashboard_reporting_db_reader, CDO.dashboard_reporting_db_reader`

Reply:
TIL!